TheFourFifths/consus-client
# Broken items report tff200

### Summary

Broken Items report page. Also, print button for the reports and some styling.

### Checklist


### How to Review

1. Start the client and server on this branch. 
2. Go to reports
3. See the faulty items report with no items.
4. Add a fault to an item.
5. Go back to the report, see the faulty item

### Links

- [TFF-200](https://msoese.atlassian.net/browse/TFF-200)
- TheFourFifths/consus/pull/104
